src/pagopa_proxy.tf

```diff
@@ -0,0 +1,147 @@
+resource "azurerm_resource_group" "pagopa_proxy_rg" {
+  count    = var.pagopa_proxy_enabled ? 1 : 0
+  name     = format("%s-pagopa-proxy-rg", local.project)
+  location = var.location
+
+  tags = var.tags
+}
+
+module "pagopa_proxy_snet" {
+  count                = var.pagopa_proxy_enabled && var.cidr_subnet_pagopa_proxy != null ? 1 : 0
+  source               = "git::https://github.com/pagopa/azurerm.git//subnet?ref=v2.0.19"
+  name                 = format("%s-pagopa-proxy-snet", local.project)
+  address_prefixes     = var.cidr_subnet_pagopa_proxy
+  resource_group_name  = azurerm_resource_group.rg_vnet.name
+  virtual_network_name = module.vnet.name
+
+  enforce_private_link_endpoint_network_policies = true
+
+  delegation = {
+    name = "default"
+    service_delegation = {
+      name    = "Microsoft.Web/serverFarms"
+      actions = ["Microsoft.Network/virtualNetworks/subnets/action"]
+    }
+  }
+}
+
+module "pagopa_proxy_redis" {
+  count                 = var.pagopa_proxy_enabled ? 1 : 0
+  source                = "git::https://github.com/pagopa/azurerm.git//redis_cache?ref=v2.0.19"
+  name                  = format("%s-pagopa-proxy-redis", local.project)
+  resource_group_name   = azurerm_resource_group.pagopa_proxy_rg[0].name
+  location              = azurerm_resource_group.pagopa_proxy_rg[0].location
+  capacity              = var.pagopa_proxy_redis_capacity
+  enable_non_ssl_port   = false
+  family                = var.pagopa_proxy_redis_family
+  sku_name              = var.pagopa_proxy_redis_sku_name
+  enable_authentication = true
+  subnet_id             = module.pagopa_proxy_snet[0].id
+
+  tags = var.tags
+
+  private_endpoint = {
+    enabled              = var.pagopa_proxy_redis_private_endpoint_enabled
+    virtual_network_id   = azurerm_resource_group.rg_vnet.id
+    subnet_id            = module.pagopa_proxy_snet[0].id
+    private_dns_zone_ids = var.pagopa_proxy_redis_private_endpoint_enabled ? [azurerm_private_dns_zone.privatelink_redis_pagopa_proxy[0].id] : []
+  }
```

Comment:
I don't know it we can put in the same subnet pagopa-proxy app service and redis private endpoint